src/components/CohortBuilder/Summary/index.js

```diff
@@ -44,11 +50,22 @@ const sortDescParticipant = (a, b) => {
 
 const CardSlot = styled(Card)`
   height: 305px;
+`;
+
+const CardSlotOverflowVisible = styled(Card)`
+  height: 305px;
   & div {
     overflow: visible;
   }
 `;
 
+const CardSlotOverflowScroll = styled(Card)`
+  height: 305px;
+  & div {
+    overflow: scroll;
+  }
+`;
```

Comment:
The card should have some build-in mechanism for scrolling, which comes with a little fade effect that goes away when at bottom. You can refer to the Saved Queries card in the dashboard page :) 
![screen shot 2019-02-08 at 5 01 33 pm](https://user-images.githubusercontent.com/11821106/52508614-39cf1300-2bc3-11e9-9ee3-c46c2ff6bb4e.png)

Reply:
Ok, fixed!